AuxTel/AuxTel-Troubleshooting/ATCS-Troubleshooting/ATMount-fails-move.rst

```diff
@@ -0,0 +1,81 @@
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Y.-AlSayyad*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *P.-Venegas*, *K.-Aubel*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Mount-Fails-to-Move:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+##########################
+AuxTel Mount Fails to Move
+##########################
+
+.. _AuxTel-Mount-Fails-to-Move-Procedure-Overview:
+
+Overview
+========
+
+This troubleshooting procedure is necessary when the AuxTel mount experiences a failure to move, specifically due to the **azimuth max velocity error exceeded**. 
+Users will observe the telescope remaining stuck with ATMCS reporting the fault.
+
+.. _AuxTel-Mount-Fails-to-Move-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+When attempting to move the telescope via point_azel or Scheduler, the faults are the followings:
+
+- The ATMCS get FAULT state ,  ATMCS provides specific error messages indicating faults in azimuth drives #1 and #2, with both fault bits activated.
+- The ATPtg get FAULT state
+
+.. error::
+    Fault event in ATMCS while in tracking enable state.  Azimuth max velocity error  exceeded.
+    /ATMCS Real-Time Application.vi/MOUNTDEV.vi/Generic Device.lvclass:Run.vi:1530004/MOUNTDEV.lvclass:NormalDeviceLogic.vi/MOUNTDEV.lvclass:ProcessDeviceLogicStateTransitions.vi/MntcTrackEnabledState.lvclass:ProcessStateTransition.vi/mount_fault_state_procedure.vi
+ 
+    Fault event in ATMCS while in tracking enable state.  Azimuth drive #2 fault bit is ON.  Azimuth drive #1 fault bit is ON.  Azimuth max velocity error  exceeded.
+    /ATMCS Real-Time Application.vi/MOUNTDEV.vi/Generic Device.lvclass:Run.vi:1530004/MOUNTDEV.lvclass:NormalDeviceLogic.vi/MOUNTDEV.lvclass:ProcessDeviceLogicStateTransitions.vi/MntcTrackEnabledState.lvclass:ProcessStateTransition.vi/mount_fault_state_procedure.vi
+
+
+.. _AuxTel-Mount-Fails-to-Move-Procedure-Procedure-Steps:
+
+
+Procedure Steps
+===============
+
+.. todo::
```

Comment:
This note is not rendering. Did you mean to show it?

Reply:
It is a note